common/src/main/java/com/microsoft/identity/common/internal/dto/AccessTokenRecord.java

```diff
@@ -319,6 +319,15 @@ public String getRefreshOn() {
         return mRefreshOn;
     }
 
+    /**
+     * Convenience method for determining if refreshIn is returned from server.
+     *
+     * @return RefreshOn is active.
+     */
+    public boolean refreshOnIsActive() {
```

Comment:
This seems confusing...refresh on vs refresh in...which one exactly are we looking for?

Reply:
I think the PR and this Javadoc seems to be expecting a refresh in from the server...but it seems the code is expecting a refresh on....am I missing something here?

Reply:
I think what I'm asking here is I'm seeking some clarification on what server actually sends us...I see this: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/014d1ad93489c48b181d7ae754e6858a70c8ebb0/common/src/main/java/com/microsoft/identity/common/internal/dto/AccessTokenRecord.java#L92-L95

and that seems like server is sending refresh_on as opposed to refresh_in?

Reply:
Yeah this was added by Logan around January I believe.. I'm not sure what he had in mind, but it might be helpful to describe the relation between refresh_in and refresh_on in that comment. I'll add it.

Reply:
The server is returning refresh_in though.